Chinese/DesertHell/OldStory/Cat2Dog/music/music.md

```diff
@@ -0,0 +1,4 @@
+說道：昨晚聽到音樂又被戳中了
+楊千嬅的“祈求天地放過一雙戀人，怕發生的永遠別發生”
+[然而..](pity/pity.md)
+[兩年前..](ago/ago.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.